eng/Versions.props

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
     <PreReleaseVersionIteration></PreReleaseVersionIteration>
+    <!-- Enable to remove prerelease label. -->
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
```

Comment:
`StabilizePackageVersion` is yet another one of these properties that existed in runtime (corefx) but aren't necessary anymore. DotNetFinalVersionKind is sufficient.

```suggestion
    <!-- Enables removing prerelease labels. -->
    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```